src/exts/tvdb_info/main.py

```diff
@@ -87,24 +89,40 @@ def __init__(self, bot: Bot) -> None:
         choices=["movie", "series"],
         required=False,
     )
+    @option("by_id", input_type=bool, description="Search by tvdb ID.", required=False)
     async def search(
-        self, ctx: ApplicationContext, query: str, entity_type: Literal["movie", "series"] | None = None
+        self,
+        ctx: ApplicationContext,
+        query: str,
+        entity_type: Literal["movie", "series"] | None = None,
+        by_id: bool = False,  # noqa: FBT001, FBT002
     ) -> None:
         """Search for a movie or series."""
         await ctx.defer()
         async with aiohttp.ClientSession() as session:
             client = TvdbClient(session)
-            match entity_type:
-                case "movie":
-                    response = await client.search(query, limit=5, entity_type="movie")
-                case "series":
-                    response = await client.search(query, limit=5, entity_type="series")
-                case None:
-                    response = await client.search(query, limit=5)
-
-        if not response:
-            await ctx.respond("No results found.")
-            return
+            if by_id:
+                match entity_type:
+                    case "movie":
+                        response = [await Movie.fetch(int(query), client, extended=True, meta=FetchMeta.TRANSLATIONS)]
```

Comment:
This could potentially cause an unhandled ValueError if query isn't `int`. Would look nicer if it was a handled error within this command, the unhandled embed from err handler mentions that those should be reported as issues.

Same for the similar line in `series` case.